### PR TITLE
gitlab issues url changed and enabled dot in namespace

### PR DESCRIPTION
**Pull request by lewicki-pk**
Hi,
Summary of changes:
- I added dot '.' in gitlab namespace (in my company we use namespaces: name.surname)
- I changed gitlab issues url to use project ID.

Second change is not so nice, but I'm new to python so forgive me that.
I didn't check tests so they may fail now due to changed url.

### CODE REVIEW

File: redmine_gitlab_migrator/gitlab.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import re
+import json
```
Comment (JocelynDelalande) [inline redmine_gitlab_migrator/gitlab.py]: Why that ?

**Comment by JocelynDelalande** [comment]
Thanks for your contribution :)

Please fix tests

Jocelyn

**Comment by JocelynDelalande** [comment]
@lewicki-pk any chance you rework the PR fixing the tests ?

**Comment by lewicki-pk** [comment]
Hi,
It was a long time ago, but I can try.
On the other hand- wouldn't that be easier for all if @Steinblock enable . (dot) in the user/project name in his #10 ?
If his code really works with gitlab 8.12 then there is no need for my stuff..
Let me know and we'll get this done

**Comment by Steinblock** [comment]
@lewicki-pk  

Yes, I can do it. I already took the most important part (changing the api to use project id) and modified the tests cases.

Anyway, I would use a broader approach

```
^(?P<base_url>https?://.*/)(?P<namespace>[^/]+)/(?P<project_name>[^/]+)/?$
`` `

see https://regex101.com/r/rQAcL6/1

**Comment by lewicki-pk** [comment]
@Steinblock 
Great.
As it comes to project and namespace naming and your regex- I just tested some strange project name and I got the message that : 
*    Name can contain only letters, digits, '_', '.', dash and space. It must start with letter, digit or '_'.
*    Path can contain only letters, digits, '_', '-' and '.'. Cannot start with '-', end in '.git' or end in '.atom'

But on the other hand the url is provided to the tool so (in my opinion) your approach would fulfill the need :+1: 

I'm closing this pull request.

Thanks

**Pull request closed without merging**